packages/core-generator/src/expressions/property-access.ts

```diff
@@ -104,6 +104,9 @@ export class PropertyAccess extends ExpressionWithExpression {
   toString(options?: toStringOptions, elements: BindingElement[] = []) {
     const member = this.getMember(options);
     if (member) {
+      if (member.isRestAttributes && options?.mutableOptions) {
+        options.mutableOptions.hasRestAttributes = true;
+      }
```

Comment:
As for now this part is refers to the Angular-generator.
So, I suggest move it to the angular-generator `PropertyAccess` class.

Reply:
it is not easy to move this code to angular PropertyAccess because in getVariableExpressions method of core generator core PropertyAccess is created and it will be hard to refactor this code and create angular PropertyAccess inside getVariableExpressions method.

Also, additional code for restAttributes is generated in react too. So, hasRestAttributes can be useful in all generators and it is not problem that hasRestAttributes is defined and used in core generator.